octobot_trading/exchanges/exchange_manager.py

```diff
@@ -308,9 +308,15 @@ def get_exchange_credentials(self, exchange_name):
             return "", "", ""
         config_exchange = self.config[common_constants.CONFIG_EXCHANGES][exchange_name]
         return (
-            configuration.decrypt_element_if_possible(common_constants.CONFIG_EXCHANGE_KEY, config_exchange, None),
-            configuration.decrypt_element_if_possible(common_constants.CONFIG_EXCHANGE_SECRET, config_exchange, None),
-            configuration.decrypt_element_if_possible(common_constants.CONFIG_EXCHANGE_PASSWORD, config_exchange, None)
+            configuration.decrypt_element_if_possible(
+                common_constants.CONFIG_EXCHANGE_KEY, config_exchange, None
+            ).strip(" "),   # remove leading and trailing whitespaces if any
```

Comment:
What if an exchange (it seems that we don't have one yet but in the future) has an api with a whitespace inside it?